Cargo.toml

```diff
@@ -136,7 +136,7 @@ winit = { version = "0.29", features = ["android-native-activity"] }
 # Metal dependencies
 block = "0.1"
 core-graphics-types = "0.1"
-metal = { version = "0.29.0" }
+metal = { version = "0.29.0", git = "https://github.com/gfx-rs/metal-rs.git", rev = "ae7030be0edff4cda88ece74137e5bcd28ef48fa" }
```

Comment:
issue: We have a release impending, and we can't release to Crates.io with Git dependencies. This needs to be resolved upstream; I'm going to investigate releasing the current mainline history of Metal to see if this can be resolved before a WGPU release.

Reply:
I have a PR open upstream: https://github.com/gfx-rs/metal-rs/pull/341

Reply:
[`metal` v0.30.0](https://crates.io/crates/metal/0.30.0) is now released!